code/modules/mining/abandoned_crates.dm

```diff
@@ -239,7 +239,7 @@
 		if(96)
 			new /obj/item/hand_tele(src)
 		if(97)
-			new /obj/item/clothing/mask/balaclava
+			new /obj/item/clothing/mask/bandit_balaclava
```

Comment:
Why are we replacing old balaclava here?

Reply:
i dont remember to be honest i will revert it